src/js/components/date-picker.js

```diff
@@ -1058,15 +1058,15 @@ const renderCalendar = (el, _dateToDisplay) => {
             class="${CALENDAR_PREVIOUS_YEAR_CLASS}"
             aria-label="Navigate back one year"
             ${prevButtonsDisabled ? `disabled="disabled"` : ""}
-          >&nbsp;</button>
+          ></button>
         </div>
         <div class="${CALENDAR_CELL_CLASS} ${CALENDAR_CELL_CENTER_ITEMS_CLASS}">
           <button
             type="button"
             class="${CALENDAR_PREVIOUS_MONTH_CLASS}"
             aria-label="Navigate back one month"
             ${prevButtonsDisabled ? `disabled="disabled"` : ""}
-          >&nbsp;</button>
+          ></button>
```

Comment:
# Date Picker JS

WHC changes the background color for all text. The `&ngsp;` causes a rectangle to appear and obstruct the icon masks:

![image](https://user-images.githubusercontent.com/25211650/159302960-9f0c8774-d07d-47d0-9203-277001c8dc3b.png)


![image](https://user-images.githubusercontent.com/25211650/159302858-f1bfddda-642c-4cc5-9f0d-335b97309020.png)

### After Fix
![image](https://user-images.githubusercontent.com/25211650/158251003-5e385868-8b5d-4d50-aeae-fb4bc9625914.png)

Reply:
Wy was this `&nbsp;` here in the first place? Does removing it cause any kind of regression in any other browser?

Reply:
I've checked Chrome, Edge, and Firefox in both standard and High Contrast modes and have yet to find an error!